packages/cli/src/decorators/Authorized.ts

```diff
@@ -3,7 +3,7 @@ import { CONTROLLER_AUTH_ROLES } from './constants';
 import type { AuthRoleMetadata } from './types';
 
 export function Authorized(authRole: AuthRoleMetadata[string] = 'any'): Function {
-	return function (target: Function | Object, handlerName?: string) {
+	return function (target: Function | object, handlerName?: string) {
```

Comment:
hmm is this expected?

Reply:
If the linter says. Apart from that it also looks correct to me. At least when I use objects as a type I use the lower case version like this.

Reply:
`object` is correct here, no idea why I wrote `Object`. Thanks for fixing this.